README.md

```diff
@@ -52,6 +52,7 @@ The register method supports the following parameters.
 |enableSPA|Boolean|Monitor the page hashchange event and report PV, which is suitable for single page application scenarios|false|false|
 |autoTracePerf|Boolean|Support sending of performance data automatically.|false|true|
 |vue|Boolean|Support vue errors monitoring|false|true|
+Notice: In tracing data requests, the service parameter is `service + '<browser>'`.
```

Comment:
This breaks the markdown. https://github.com/apache/skywalking-client-js/blob/0980bf66bf60d6a83a782b6aca9836b35b71755b/README.md

You could add this into the `service` row directly.